.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Summary of the Pull Request
+
+## Detailed Description of the Pull Request / Additional comments
+
+## Example Log Event (In Case of FP Fixes)
```

Comment:
I don't want to use the word mandatory for FP, but having it for a new rule, or an adjustment is good too.

Reply:
I would say It is mandatory for new contributors fixing FP to provide some logic behind it. Since we can't accept FP filtering blindly.
But for obvious FPs or from trusted sources then it's fine.

Reply:
where do I see what fields are mandatory?

Reply:
There is nothing mandatory. This file will be added automatically once you create a PR and you should fill it in as necessary.

Reply:
Then I don't see a problem. We can decide on a case-by-case base if we accept the PR or need more input.

Reply:
Yes of course. This is just to get more conform PRs in the future